### DISCUSSION TITLE
How do we know they will not re-purpose plot space to be the worlds largest porn server?

### DISCUSSION THREAD

**Discussion by room101-dev** [Chia-Network/chia-blockchain#5777] [General]
One thing strange about CHIA is we generate these 108gb plots, that except for the file name contain nothing of use. What's stopping some clever programmer at the mining end, and I think this is why CHIA-DEV hates HPOOL (CHINA) because they will beat the chia-dev at their own game.

So you pool, and your give your plot access to a pool-miner, what's to stop the miner from writing content back to your plot in the future. Not a damn thing, your running their client, and it can do any damn thing it wants. Sure you could setup all your plots read-only and hope for the best.

With the so called pool-protocol, and first told we could keep our plots forever, now told to delete our plots, or even told to use the new portable-plot format when it comes out, we know nothing about this portable-format. How do we not know the plots will be used like file-coin, where people pay for storage? We have no idea, no wonder they want everybody to quit using hpool, and to only  use their pool

In the future these pools and their mining software, and especially if the pools don't give out 'open-source' you have no idea of what they're actually doing, if for instance they're uploading CP to your computer as encrypted files, you would never know.

Bittorrent is infamous as being the largest use of the internet on earth, and of course the majority of BT is porn, and a lot of that is CP. The very person who created BT created chia, you can't say they don't think about these things. Because its  their business for the past 10+ years.

I know the Chinese Gov is very interested in CHIA, from a standpoint like file-coin of having general storage that is not under the control of the west like on google or amazon ( both cia/nsa client companys ); China now controls HPOOL, and majority of plots and certainly all the IOU cash infused in CHIA(XCH) came from China, the west hasn't contributed shit towards CHIA.

Just  like BITCOIN is owned by China, CHIA is owned by China, but you'll never hear anybody admit that BITCOIN is owned by China because most people are idiots, or just don't have a clue that 90% of all mining, and mining-pools on earth for BTC are ran out of CHINA, just like CHIA now.

China will not use CHIA storage for PORN or CP, they'll just use it as a global storage that can't be controlled by CIA or NSA, of course that drives chia-dev crazy, because in order to be a NYSE IPO you must bend over and be owned by CIA, thus Hoffman&Cohen may have already sold out their soul, the Chinese will never bend over to the CIA.

**Comment by sonpython** [comment]
`With the so called pool-protocol, and first told we could keep our plots forever, now told to delete our plots, or even told to use the new portable-plot format when it comes out, we know nothing about this portable-format. `

agree 

The Chia Team is not consistent. They change the whole network plots format  >8000 PiB of storage just for what they call portable-plot. How many time they will change it again?

**Comment by FSys77** [comment]
I am OK with storing porn. _Anything_ is more meaningful than this crap that is currently in the plot files.

**Comment by ByronAP** [comment]
Just reading this now in October 2021 (almost 6 months later) and China has banned "BITCOIN" as well as ALL other public crypto currencies. I wonder if the OP @room101-dev has any more silly theories? 

Ref: https://www.bbc.com/news/technology-58678907

**Reply by cypher619** [reply]
I agree.  This is a massive exodus of mining equiptment from china to other countries around the world.  Chinese govt is even scouring the countryside and hunting down farms that are ignoring the orders.

**Comment by BrandtH22** [comment]
@room101-dev , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
